ui/transactions_page.go

```diff
@@ -0,0 +1,277 @@
+package ui
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"time"
+
+	"gioui.org/f32"
+	"gioui.org/layout"
+	"gioui.org/op"
+	"gioui.org/op/paint"
+	"gioui.org/text"
+	"gioui.org/unit"
+	"gioui.org/widget"
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/raedahgroup/dcrlibwallet"
+	"github.com/raedahgroup/godcr/ui/decredmaterial"
+	"github.com/raedahgroup/godcr/wallet"
+	"golang.org/x/exp/shiny/materialdesign/icons"
+)
+
+const (
+	PageTransactions                            = "txs"
```

Comment:
there's no need to abbreviate transactions. You can have it as "transactions", that way all pages follow the same convention.